app/controllers/experience_levels_controller.rb

```diff
@@ -0,0 +1,59 @@
+class ExperienceLevelsController < ApplicationController
+  before_action :check_user_is_admin
```

Comment:
This method could totally be on `ApplicationController` or we could create a `Concern`